plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveWriterFactory.java

```diff
@@ -636,22 +638,34 @@ public HiveWriter createWriter(Page partitionColumns, int position, OptionalInt
                 hiveWriterStats);
     }
 
```

Comment:
I think you could add unit test for it in `io.trino.plugin.hive.TestHiveWriterFactory` if you make this method staic and package private (then move this method to bottom of class)